testing/scenario_app/bin/android_integration_tests.dart

```diff
@@ -178,7 +193,11 @@ Future<void> _run({
         await skiaGoldClient!.auth();
         log('skia gold client is available');
       } else {
-        log('skia gold client is unavailable');
+        if (useSkiaGold) {
+          panic(<String>['skia gold client is unavailable']);
+        } else {
+          log('skia gold client is unavaialble');
```

Comment:
Should this be a bit louder since they are requesting it but it will fail?  "Error: Skia gold's client, goldctl, is unavailable"?

Reply:
No. I added a clarification, this is in the non-CI case where it could be expected not to have Skia gold hooked up.